benchmarks/README.md

```diff
@@ -0,0 +1,22 @@
+# ACCP Benchmarks
+
+## Running the benchmarks
+
+The benchmarks can use locally built ACCP or published ACCP.
+
+### Benchmarking published ACCP to Maven
+
+```bash
+./gradlew lib:jmh
+```
+
+* `-Pfips` flag can be used to use the FIPS artifacts for benchmarking.
+* `-PaccpVerion="ACCP_VERSION"` can be used to run benchmarks for a specific version of ACCP
+
+### Benchmarking locally built ACCP
+
+Use `-PaccpLocalJar="PATH_TO_LOCAL_JAR"`:
+
+```bash
+ ./gradlew -PaccpLocalJar="../../build/cmake/AmazonCorrettoCryptoProvider.jar" lib:jmh
```

Comment:
minor nit: can we also specify the `jmhReport` task in this example?

Reply:
Modified the build.gradle.kts so that the `jmhReport` would always be executed after `jmh`. In this way, we don't have to explicitly run it.